config/routes.rb

```diff
@@ -131,6 +131,12 @@
     end
   end
 
+  namespace :admin do
```

Comment:
I'm thinking this should follow the pattern above, both in terms of namespacing `StatsAdmin` as `Sufia::StatsAdmin` and in terms of checking to make sure that is defined first.